File: elias/Problema0249/problema0249.java
import java.util.Scanner;

public class problema0249 {
    public static void main(String[] args) {
        Scanner input = new Scanner (System.in);
        int[] A = new int[10];
        int[] B = new int[10];
        for(int i=0;i<10;i++){
            System.out.print("Digite um número inteiro para a posição "+i+" do vetor: ");
            A[i] = Integer.valueOf(input.nextLine());
        }
        for(int i=0;i<10;i++){
            int somatorio = 0;
            for(int j=1;j<=A[i];j++){
                somatorio += j;
                B[i] = somatorio;
            }
        }
        System.out.println("Vetor B:");
        for(int i=0;i<10;i++){
            System.out.println(B[i]);
        }
    }
}

File: elias/Problema0249/problema0249.py
A = []
B = []
for i in range (10):
    n = int(input("Digite um número inteiro para a posição %d do vetor:  " %i))
    A.append(n)
for i in range (10):
    somatorio = 0
    for j in range (1, A[i] + 1, 1):
        somatorio += j
    B.append(somatorio)
print("Vetor B:")
for i in range (10):
    print(B[i])